### PR TITLE
feat: publish prerelease to npm

### PR DESCRIPTION
**Pull request by tjwelde**
## fixes KILTProtocol/ticket#851
Please include a summary of the changes provided with this pull request and which issue has been fixed. 
Please also provide some context if necessary.

## How to test:
Please provide a brief step-by-step instruction.
If necessary provide information about dependencies (specific configuration, branches, database dumps, etc.)

- Step 1
- Step 2
- etc.

## Checklist:

- [ ] I have verified that the code works
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/boy-scout-rule/)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: .github/workflows/devpackpublish.yml [hunk 1]
```diff
@@ -3,25 +3,45 @@ on:
   push:
     branches:
       - develop
+      - tw-851-versioning
```
Comment (rflechtner) [inline .github/workflows/devpackpublish.yml:6]: remove before merging pls

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kiltprotocol/sdk-js",
-  "version": "0.19.0",
+  "version": "0.19.1-3",
```
Comment (rflechtner) [inline package.json:3]: this will probably remain bc these prereleases are already published, right?

**Review by rflechtner — APPROVED** [review]
LGTM!

File: package.json (hunk 2 above)
Comment (tjwelde) [reply package.json:3]: yes

**Pull request merged**